ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/exception/InvalidAuthenticationException.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package eu.elixir.ega.ebi.dataedge.config;
+package eu.elixir.ega.ebi.dataedge.exception;
 
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
```

Comment:
@anandmohan777 , I've created (pending to push) InvalidAuthenticationException in the ega-data-api-commons because I will be using it too for Htsget. In order to do not duplicate code, can you move this class there? I'm more than happy to use yours :) It's important that we try to reuse as much code as we can!